src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -275,6 +267,53 @@ private static boolean changeIsLargerThanThreshold(
             ||  deltaBwe < -1 * previousBwe * Config.bweChangeThresholdPct() / 100;
     }
 
+    static class EndpointMultiRank
+    {
+        final int speakerRank;
+        final EndpointConstraints endpointConstraints;
+        final AbstractEndpoint endpoint;
```

Comment:
Can you document which endpoint this refers to? Is it the one for whom the constraints apply (the sender)? Or the one applying the constraint (the receiver)?